docs/api-reference/log/stats.md

```diff
@@ -18,9 +19,10 @@ executionTime.timeEnd();
 
 ### constructor
 
-`new Stats({id})`
+`new Stats({id, stats})`
 
 * `id` (`String`) - the id of the `Stats` object.
+* `stats` (`String`) - the list of stats. Each element in the stats could be either`Stat` object or `{id, type}` (type is optional, default is `count`);
```

Comment:
Nit: `stats` type: `String[] | Object[]` or `(String | Object)[]`

An alternative is to make type an `Object` map with `id`s as keys. This way we could separating `id` (easy to use in code) from `name` (being a printable, human readable name to use in the stats widget).

```js
new Stats({
  id, 
  stats: {
    count: {name: 'Count', type: 'count', resetOnUpdate: true}
  }
});
```